### ISSUE TITLE
Llama-2 is out, commercial-use WizardLM

### ISSUE THREAD

**Issue by ricklamers** [nlpxucan/WizardLM#131]
WizardLM is amazing! Now that Llama-2 is out & available for commercial use, will we get a commercial-use available WizardLM?

**Comment by ricklamers** [comment]
>You will not use the Llama Materials or any output or results of the Llama Materials to improve any other large language model (excluding Llama 2 or derivative works thereof).

Oh nevermind 🤷

**Comment by ricklamers** [comment]
From: https://github.com/facebookresearch/llama/blob/main/LICENSE

**Comment by ricklamers** [comment]
>or derivative works thereof

Perhaps this is an out? I'm no lawyer 😅

**Comment by cornleaf** [comment]
Yes ,same question to author , maybe you should change the license asap, then the project will get more attetion
